src/components/Wallet/Wallet.tsx

```diff
@@ -40,7 +40,7 @@ export default function Wallet(props: WalletProps) {
     const encodedTransaction = algosdk.encodeUnsignedTransaction(transaction);
     const signedTransactions = await signTransactions([encodedTransaction]);
 
-    const { id } = await sendTransactions(signedTransactions);
+    const { id } = await sendTransactions(signedTransactions, 4);
```

Comment:
Even though Intellisense will show the argument name/type in an IDE, using a variable improves readability.

```suggestion
    const waitRoundsToConfirm = 4
    const { id } = await sendTransactions(signedTransactions, waitRoundsToConfirm);
```